packages/shopping/src/application.ts

```diff
@@ -101,6 +101,16 @@ export class ShoppingApplication extends BootMixin(
 
     this.bind(UserServiceBindings.USER_SERVICE).toClass(UserManagementService);
     this.add(createBindingFromClass(SecuritySpecEnhancer));
+
+    // Use env variable JWTSECRET as secret key for JWT
+    const token = process.env.JWTSECRET;
+    if (!token) {
```

Comment:
Thanks for the PR. I don't think its necessary to log an error rather just check that the `JWT_SECRET` exists and if it does use so that for those who dont want to set the environment variable are not affected by this change

Reply:
The error message is there because I think using a publicly available key for signing JWTs is a very bad idea.
In production code I would have terminated the process if JWTSECRET was not set.

Reply:
Interesting! maybe throwing an error would be more ideal based on what you mentioned above. Am just thinking if the variable is not set then it defaults back to the current behaviour where we use the default token?

Reply:
Yes, if JWTSECRET is not set, you will get an error message, but other than that, it will work as before.
Changing to aborting program if JWTSECRET is not set will most probably break existing tests,
so I would not want to do that now.

Reply:
@johanschvn what about generating a secret via `crypto.randomBytes` if the env variable is not set?

If you run a single instance of the application this would be the most secure approach, however if multiple replicas of the app are used you will have to set the secret an env variable because all instances need the same secret

Reply:
> If you run a single instance of the application this would be the most secure approach, however if multiple replicas of the app are used you will have to set the secret an env variable because all instances need the same secret

Thanks @nflaig. @johanschvn I think to keep things simple given the nature of the project you can generate a random string and leave a comment based on the above

Once these changes are done it will be ready to land on my side, again, thanks for the contributions